cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/upgrade/operation/AwsRdsVersionOperations.java

```diff
@@ -0,0 +1,108 @@
+package com.sequenceiq.cloudbreak.cloud.aws.connector.resource.upgrade.operation;
+
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.amazonaws.services.rds.model.DescribeDBEngineVersionsRequest;
+import com.amazonaws.services.rds.model.DescribeDBEngineVersionsResult;
+import com.amazonaws.services.rds.model.UpgradeTarget;
+import com.sequenceiq.cloudbreak.cloud.aws.common.client.AmazonRdsClient;
+import com.sequenceiq.cloudbreak.cloud.exception.CloudConnectorException;
+import com.sequenceiq.cloudbreak.cloud.model.DatabaseEngine;
+import com.sequenceiq.cloudbreak.common.database.Version;
+import com.sequenceiq.cloudbreak.util.VersionComparator;
+
+@Service
+public class AwsRdsVersionOperations {
+
+    static final String ENGINE_VERSION = "engineVersion";
+
+    private static final Pattern ENGINE_VERSION_PATTERN = Pattern.compile("^(\\d+)(?:\\.\\d+)?$");
+
+    private static final int GROUP_MAJOR_VERSION = 1;
+
+    private static final int VERSION_9 = 9;
+
+    private static final int VERSION_13 = 13;
+
+    private static final String POSTGRES = "postgres";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AwsRdsVersionOperations.class);
+
+    public String getDBParameterGroupFamily(DatabaseEngine engine, String engineVersion) {
+        switch (engine) {
+            case POSTGRESQL:
```

Comment:
I merely moved this code from another place.
I know some code in Cloudera uses MySql, but no idea if it will be a requirement in CDP. I personly have not heard of it, I will ask.

Reply:
Not in the near future. I converted it to if-else.